app/assets/javascripts/modules/base-target.js

```diff
@@ -0,0 +1,26 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function BaseTarget () { }
+
+  BaseTarget.prototype.start = function () {
+    if (this.pageIsInIframe()) {
+      this.setBaseTarget()
+    }
+  }
```

Comment:
Is it conventional for us to have a `start()` method? I wonder, see we immediately invoke it, whether this should just be done in the constructor?

Reply:
I tried to align this with the current module pattern that we have in different scripts across `static` (regardless of its complexity - as this could potentially be extended)